### ISSUE TITLE
Missing parameters description after merging

### ISSUE THREAD

**Issue by lednhatkhanh** [Urigo/merge-graphql-schemas#84]
I just realized that after merging all my types, the mutation and query's parameter descriptions are gone.

For example:
```graphql
type Mutation {
  # Register new user, need to remove Authorization and RefreshToken headers
  register(
    # User firstname
    firstname: String!
    # User lastname
    lastname: String!
    # User email address
    email: String!
    # User password
    password: String!
  ): User!
}
```

becomes this:
```graphql
type Mutation {
  # Register new user, need to remove Authorization and RefreshToken headers
  register(
    firstname: String!
    lastname: String!
    email: String!
    password: String!
  ): User!
```

Please fix this asap, thank you!

**Comment by RichardLitt** [comment]
Thanks @lednhatkhanh. This may be intended functionality. It might be a good idea to add an option for keeping comments when merged.

Would you be interested in helping us make a PR for this? If not, it may take some time before this can be added.

**Comment by lacolaco** [comment]
I've tried to fix this problem, but it's difficult a bit.
A essencial problem is that graphql's `getDescription` utitily cannot treat arguments' comments. Always it returns `undefined`. So I think there's nothing we can do on `merge-graphql-schemas` 's layer.

**Comment by RichardLitt** [comment]
@cfnelson Any ideas?

**Comment by lacolaco** [comment]
@RichardLitt I've sent an issue about that to graphql-js repo.

https://github.com/graphql/graphql-js/issues/1281

Please +1 on it!

**Comment by sp00m** [comment]
Looks like the linked issue has been closed (fixed?). Can `merge-graphql-schemas` now handle these comments properly?

**Comment by kamilkisiela** [comment]
#196 should fix it

**Comment by kamilkisiela** [comment]
It's available now under `v1.6.0-beta.2`. Let us know if it works :)
